inc/saiacl.h

```diff
@@ -227,17 +241,47 @@ typedef enum _sai_acl_table_group_attr_t
     SAI_ACL_TABLE_GROUP_ATTR_START,
 
     /**
-     * @brief Dummy attribute
+     * @brief ACL stage
+     *
+     * @type sai_acl_stage_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_STAGE = SAI_ACL_TABLE_GROUP_ATTR_START,
+
+    /**
+     * @brief List of ACL bind points where this group will be applied
+     *
+     * @type sai_acl_bind_point_t 
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_BIND_POINT,
+
+    /**
+     * @brief Priority
      *
-     * @type sai_int32_t
-     * @flags READ_ONLY
+     * Value must be in the range defined in
+     * [SAI_SWITCH_ATTR_ACL_TABLE_GROUP_MINIMUM_PRIORITY,
+     * SAI_SWITCH_ATTR_ACL_TABLE_GROUP_MAXIMUM_PRIORITY]
+     *
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
      */
-    SAI_ACL_TABLE_GROUP_ATTR_DUMMY = SAI_ACL_TABLE_GROUP_ATTR_START,
+    SAI_ACL_TABLE_GROUP_ATTR_PRIORITY,
```

Comment:
@lguohan based on DELL's feedback the idea was to use this GROUP priority to resolve parallel lookups in some cases that ASIC may not support parallel lookups based on various rules. And in that case the list of GROUPs that are bound may need some priority. 

Also, i made it optional and not mandatory on create. What do you think?